corehq/util/context_processors.py

```diff
@@ -170,7 +170,7 @@ def get_demo(request):
     num_trial_days_remaining = 0
     if (settings.IS_SAAS_ENVIRONMENT
             and settings.ANALYTICS_IDS.get('HUBSPOT_API_ID')):
-        if not request.user.is_authenticated:
+        if hasattr(request, 'user') and not request.user.is_authenticated:
```

Comment:
Might want to use ```getattr(request, 'user', None)``` because in principal it's possible that ```request.user == None``` and that would also error.

Reply:
Good call, thanks!